eos/maths/polylog.hh

```diff
@@ -3,7 +3,7 @@
 #ifndef EOS_GUARD_EOS_UTILS_POLYLOG_HH
 #define EOS_GUARD_EOS_UTILS_POLYLOG_HH 1
```

Comment:
You changed the name of the flag for some files, but not all of them. Is there a reason for that?

Reply:
I changed the ones I saw. Went through all files and fixed all include guards.